exotica_core/include/exotica_core/kinematic_tree.h

```diff
@@ -243,7 +259,11 @@ class KinematicTree : public Uncopyable
     void UpdateJdot();
 
     // Joint limits
+    // TODO: Add effort limits
     Eigen::MatrixXd joint_limits_;
+    Eigen::VectorXd velocity_limit_;
```

Comment:
If it's a vector, then we need plural again :-) I meant singular for the individual joint limit, but plural for the vector that concatenates them